dev/scripts/watch-metrics.sh

```diff
@@ -101,12 +101,6 @@ if [[ -z "${host}" || -z "${port}" || -z "${metrics_file_prefix}" || -z "${metri
     exit 1
 fi
 
-# check if command `nslookup` exists
-if ! command -v nslookup > /dev/null 2>&1; then
-  echo "ERROR: nslookup command not found"
-  exit 1
-fi
-
```

Comment:
Thanks for removing this as per @dbw9580 's comment. For some reason I don't see this change when reviewing the cumulative changes so just make sure this is still there.

Reply:
Maybe my previous review was not clear enough -- I meant this check for the existence of the hostname is too restrictive: https://github.com/Alluxio/alluxio/blob/1c2a20c53bba421e58d2e6efec8f31af9bb8829c/dev/scripts/watch-metrics.sh#L104-L108

@bobbai00 You seem to have removed the check for the existence of the `nslookup` command. That is fine, IMO.

Reply:
Ok I see. So I need to change the way of checking the existence of `nslookup` right?

Reply:
Checking the existence of `nslookup` is fine. You can revert this commit 46e36c9df1ad8bddfd6788517b63cc701a565476. Then remove these lines https://github.com/Alluxio/alluxio/blob/1c2a20c53bba421e58d2e6efec8f31af9bb8829c/dev/scripts/watch-metrics.sh#L104-L108

Reply:
Got it.